packages/suite-native/src/components/suite/Text/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
shoudln't this be just a `<P>` in `@trezor-components` ?

Reply:
Forgotten experiment, I've moved it to components lib in a fixup commit.